lib/linear_percent_indicator.dart

```diff
@@ -346,6 +354,12 @@ class _LinearPainter extends CustomPainter {
     _paintLine.color = progress.toString() == "0.0"
         ? progressColor.withOpacity(0.0)
         : progressColor;
+
+    if (progressBorderColor != null) {
+      _paintLineBorder.color = progress.toString() == "0.0"
```

Comment:
can you reuse/refactor this condition?  `progress.toString() == "0.0"`

Reply:
I just copy from line 354, but I'll refactor